config/i18n/locales/english/translations.json

```diff
@@ -8,8 +8,8 @@
     "label": "Search",
     "accessible-name": "Submit your search query",
     "placeholder": {
-      "default": "Search our tutorials",
-      "numbered": "Search {{ roundedTotalRecords }}+ tutorials"
+      "default": "Search our news articles, tutorials and books",
+      "numbered": "Search {{ roundedTotalRecords }}+ news articles, tutorials and books"
     },
     "see-results": "See all results for {{ searchQuery }}",
     "no-tutorials": "No tutorials found"
```

Comment:
To make it consistent with the /learn, this could be changed to `no-results` and text to `"No results found"`. `src/_includes/assets/js/search-bar.js` also would need to be changed.

Reply:
This should also go in.